GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/NetworkRequestController.cs

```diff
@@ -49,24 +49,22 @@ public static void LoadGameOnline()
         }
     }
 
-    public static ServerResponseModel verifyAndSaveUserPurchase(string purchaseToken)
+    public static void verifyAndSaveUserPurchase(Product product)
     {
         var values = new Dictionary<string, string>
         {
-            ["purchaseToken"] = purchaseToken
+            ["receipt"] = getReceiptDetails(product.receipt),
+            ["isSubscription"] = product.definition.id.Contains("subscription").ToString()
```

Comment:
Have this check at server, do not do it at client. Just send receipt is good enough.

Reply:
Yep, will do 👍